Package.swift

```diff
@@ -36,7 +36,7 @@ let package = Package(
         .package(
             name: "SnapshotTesting",
             url: "https://github.com/pointfreeco/swift-snapshot-testing.git",
-            .upToNextMajor(from: "1.8.0")
+            .upToNextMajor(from: "1.10.0")
```

Comment:
What's the advantage of this change? Since we're using `upToNextMajor(from:)`, consumers should still be able to use 1.10.x in their projects.

Reply:
I suppose the more critical aspect of this change is the increase in minimum platform change (from iOS 12 to iOS 13) which is currently a resolution failure. 

I'll revert this part as functionally it should be the same.